_sass/blocks/how-it-works/_subpage_diving_graphics.scss

```diff
@@ -206,17 +206,17 @@
 
     @include respond-to(medium-up) {
       background: url('../img/how-it-works/coal-top.png') top left no-repeat, url('../img/how-it-works/coal-bottom.png') bottom left no-repeat;
-      background-size: 100%, 100%;
+      background-size: 102%, 102%;
```

Comment:
Properties should be ordered background, background-position-x, background-size, height, position, z-index